station.rb

```diff
@@ -0,0 +1,26 @@
+class Station
+  attr_reader :name, :trains
+
+  def initialize(name)
+    @name = name
+  end
+
+  def take_train(train)
+    @trains = [train]
+  end
+
+  def freight
+    @trains.count {|x| x.type == 'freight'}
```

Comment:
```suggestion
    @trains.count { |train| train.type == 'freight' }
```
рекомендуется по возможности использовать переменные с полноценными именами